apps/condo/domains/notification/constants/constants.js

```diff
@@ -334,6 +334,11 @@ const MESSAGE_STATUSES = [
     MESSAGE_CANCELED_STATUS,
 ]
 
+const DEVICE_PLATFORM_ANDROID = 'android'
+const DEVICE_PLATFORM_IOS = 'ios'
+const DEVICE_PLATFORM_WEB = 'web'
+const DEVICE_PLATFORM_TYPES = [DEVICE_PLATFORM_ANDROID, DEVICE_PLATFORM_IOS, DEVICE_PLATFORM_WEB]
```

Comment:
please add `unknown` or something like `integration`? for server-side partner integrations. Should they use web?

Reply:
they should use one that fits their platform. we can ad different platforms like `winmobile` for example.